### PR TITLE
gallery show component

### PR DESCRIPTION
**Pull request by Abhay2012**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ const routes: Routes = [
   { path: 'en/cities',  loadChildren: './CitiesComponent/cities.module#CitiesModule' },
   { path: 'en/reports', loadChildren: './ReportsComponent/reports.module#ReportsModule' },
   { path: 'en/videos',  loadChildren: './VideosComponent/videos.module#VideosModule' },
+  { path: 'en/galleries', loadChildren: './GalleriesComponent/galleries.module#GalleriesModule' },
```
Comment (piousbox) [inline src/app/app-routing.module.ts]: is this needed? I don't think there is an index of galleries. Where does this route? Remove it if it's unused

File: src/app/GalleriesComponent/GalleriesShow.html [hunk 2]
```diff
@@ -1,15 +1,34 @@
...
+    <!-- <p>message: {{gallery.message}}</p>
+        <p>description: {{gallery.description}}</p> -->
+    <div class="gallery">
+      <div class='card' *ngFor="let ph of gallery.photos">
+        <div class='card-inner'>
+          <ion-img [src]=ph.thumb_url></ion-img>
+        </div>
+      </div>
+    </div>
+    <!-- Horizontal Image -->
```
Comment (piousbox) [inline src/app/GalleriesComponent/GalleriesShow.html]: It should should all images, one after another, not just 5th and then 1st.

File: src/app/CitiesComponent/CitiesShow.page.html [hunk 3]
```diff
@@ -27,14 +27,14 @@ <h1>{{ city.name }}</h1>
           </div>
```
Comment (piousbox) [inline src/app/CitiesComponent/CitiesShow.page.html]: additionally, can you make the cursor change to "pointer" on hover of newsitem's title or picture in the newsitem? It doesn't look like it's clickable right now.

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (Abhay2012) [reply src/app/app-routing.module.ts]: Yes, this is required, this route will load GalleriesModule and its routing

File: src/app/CitiesComponent/CitiesShow.page.html (hunk 3 above)
Comment (Abhay2012) [reply src/app/CitiesComponent/CitiesShow.page.html]: done

File: src/app/GalleriesComponent/GalleriesShow.html (hunk 2 above)
Comment (Abhay2012) [reply src/app/GalleriesComponent/GalleriesShow.html]: done

**Pull request merged**